tests/openvino/test_modeling.py

```diff
@@ -601,6 +602,20 @@ def test_pipeline(self, model_arch):
         del model
         gc.collect()
 
+    @parameterized.expand(SUPPORTED_ARCHITECTURES)
+    def test_sentence_tenaformers_pipeline(self, model_arch):
```

Comment:
small typo 
```suggestion
    def test_sentence_transformers_pipeline(self, model_arch):
```

also we can also remove pipeline for clarity as it's used for other tests to refer to the transformers pipelines 

```suggestion
    def test_sentence_transformers(self, model_arch):
```